pages/programs/baton_roots.html

```diff
@@ -1 +1,38 @@
 
+<body>
+	<div id="header">
+		<div class="grid-st">
+			<img src="http://www.thewallsproject.org/img/shared/walls_logo.svg" width="45%" />
+		</div>
+		<div class="grid-end">
+			<div id="nav">
+				<div class="nav-1"><a href="#">Progress<span class="u"></span></a></div>
```

Comment:
Href's not pointing to page sections. We'll add this after all sections are completed.